sql/ddl/alter/add-column.md

```diff
@@ -82,14 +82,16 @@ DROP COLUMN boring_column;
 ---
 ## Practice
 
-Consider the "gender" table. We want to add a new column called "members". This column will store data about how many members each gender has:
+Consider the `"gender"` table. We want to add a new column called `"members"`. This column will store data about how many members each gender has.
```

Comment:
In the case of `"gender"` vs `gender`, in this scenario `gender` would win. The reason is that `"this"` implies that `this` is a string type of sorts while `this` just implies it's some sort of emphasized programming aka code-related thingy and nothing more, which feels more suitable for table names.

```suggestion
Consider the `gender` table. We want to add a new column called `members`. This column will store data about how many members each gender has.
```

EnkiCamp content does the same, right?

<img width="1045" alt="Screen Shot 2019-12-04 at 10 49 09 AM" src="https://user-images.githubusercontent.com/9661806/70157532-c16e9c80-1683-11ea-95f6-b522dd0c9fb4.png">

Reply:
agreed, this just got caught in the grep & replace when it shouldn't have.